libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -242,11 +242,11 @@ data Permissions = Permissions
 
 data Perm =
       CreateConversation
-    | DeleteConversation
+    | DeleteConversation  -- NOTE: This gets now overruled by conv level checks
     | AddTeamMember
     | RemoveTeamMember
-    | AddRemoveConvMember
-    | ModifyConvMetadata
+    | AddRemoveConvMember -- NOTE: This gets now overruled by conv level checks
+    | ModifyConvName      -- NOTE: This gets now overruled by conv level checks
```

Comment:
How about you rename all of these to `NotInUseDeprecatedXX` or remove them entirely?

Reply:
I renamed them because removing them will break clients.